app/views/devise/sessions/new.html.erb

```diff
@@ -10,21 +10,34 @@
       margin_bottom: 3,
     } %>
 
-    <%= form_with(url: user_session_path) do %>
+    <%= form_with(url:new_user_session_path) do %>
       <% if resource %>
         <%= render "devise/shared/error_messages", resource: resource %>
       <% end %>
 
-      <%= hidden_field_tag :"user[email]", @login_state.user.email %>
+      <% if params[:jwt] %>
+        <%= hidden_field_tag :jwt, params[:jwt] %>
+      <% end %>
 
       <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: t("devise.sessions.new.fields.email.label"),
+        },
+        name: "user[email]",
+        type: "email",
+        id: "email",
+        value: @email,
+        error_message: devise_error_items(:email, @resource_error_messages),
+        autocomplete: "email",
```

Comment:
Consider: using `username` over `email` for better support with password managers - detail in main comment
```suggestion
        autocomplete: "username",
```